src/CabanaPD_Solver.hpp

```diff
@@ -255,14 +253,8 @@ class SolverElastic
             // Update ghost particles.
             comm->gatherDisplacement();
 
-            // Do not need to recompute LPS weighted volume here without damage.
-            // Compute/communicate LPS dilatation (does nothing for PMB).
-            force->computeDilatation( *particles, *neighbors,
-                                      neigh_iter_tag{} );
-            comm->gatherDilatation();
-
-            // Compute internal forces.
-            computeForce( *force, *particles, *neighbors, neigh_iter_tag{} );
+            // Update force and related quantities.
```

Comment:
Do we need the same comment "// Update force and related quantities." before Line 200, etc.

Reply:
It currently says "Compute initial forces" - push a change if you'd like